cellrank/pl/_graph.py

```diff
@@ -174,7 +173,7 @@ def plot_arrows(curves, G, pos, ax, edge_weight_scale):
                 X, Y = X[::-1], Y[::-1]
 
             mid = len(X) // 2
-            posA, posB = zip(X[mid : mid + 2], Y[mid : mid + 2])  # noqa
```

Comment:
What does the # noqa do?

Reply:
It's used to silence flake8

Reply:
Okay, that's what I was guessing. I suppose the new tags silence only particular flake8 complaints.